src/dali_backend.cc

```diff
@@ -426,23 +429,29 @@ TRITONBACKEND_ModelInstanceExecute(
     catch (std::runtime_error& e) {
       LOG_MESSAGE(TRITONSERVER_LOG_ERROR, (e.what()));
       error = TRITONSERVER_ErrorNew(
-          TRITONSERVER_Error_Code::TRITONSERVER_ERROR_UNKNOWN,
-          make_string("runtime error: ", e.what()).c_str());
+              TRITONSERVER_Error_Code::TRITONSERVER_ERROR_UNKNOWN,
+              make_string("runtime error: ", e.what()).c_str());
     }
     catch (...) {
       LOG_MESSAGE(TRITONSERVER_LOG_ERROR, ("Unknown error"));
       error = TRITONSERVER_ErrorNew(
-          TRITONSERVER_Error_Code::TRITONSERVER_ERROR_UNKNOWN,
-          "Unknown DALI Backend error");
+              TRITONSERVER_Error_Code::TRITONSERVER_ERROR_UNKNOWN,
+              "Unknown DALI Backend error");
     }
 
+    exec_end_ns = detail::capture_time();
+
+    RETURN_IF_ERROR(TRITONBACKEND_ModelInstanceReportStatistics(
```

Comment:
You should also report the overall batch statistics, like here: https://github.com/triton-inference-server/onnxruntime_backend/blob/main/src/onnxruntime.cc#L1135

Reply:
Done